cloud-storage-uploader/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Cloud Storage Uploader
+description: Upload files or folders to GCS buckets
+author: GoogleCloudPlatform
+inputs:
+  service_account_key:
+    description: |-
+      Optional GCS service account credentials as JSON or base64 encoded string
```

Comment:
Don't be afraid to be verbose with these descriptions. Once GitHub marketplace supports multiple actions per repo, these will show up there.

Reply:
Added more verbose descriptions PTAL